Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -3741,6 +3742,7 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         case CSSPropertyInsetInline:
         case CSSPropertyInsetInlineEnd:
         case CSSPropertyInsetInlineStart:
+        case CSSPropertyJustifyTracks:
```

Comment:
Please file a bug to add animation support for this property and reference it here with a `FIXME`.

Reply:
That said, I expect this single line would do:

`new DiscretePropertyWrapper<JustifyTracks>(CSSPropertyJustifyTracks, &RenderStyle::justifyTracks, &RenderStyle::setJustifyTracks)`

… and make sure there is WPT coverage for this, if it doesn't exist.